doc/source/serve/performance.md

```diff
@@ -20,20 +19,79 @@ We are continuously benchmarking Ray Serve. The metrics we care about are latenc
+     - Head node: AWS EC2 m5.8xlarge. 32 worker nodes: AWS EC2 m5.8xlarge.
+     - * per_thread_latency_avg_ms = 0.0
+       * per_thread_latency_max_ms = 0.0
+       * per_thread_avg_tps = 0.0
+       * per_thread_max_tps = 0.0
+       * per_node_avg_tps = 0.35
+       * per_node_avg_transfer_per_sec_KB = 0.05
+       * cluster_total_thoughput = 6964
+       * cluster_total_transfer_KB = 1047.28
+       * cluster_total_timeout_requests = 6964.0
+       * cluster_max_P50_latency_ms = 0.0
+       * cluster_max_P75_latency_ms = 0.0
+       * cluster_max_P90_latency_ms = 0.0
+       * cluster_max_P99_latency_ms = 0.0
+   * - `Deployment Graph: Ensemble <https://github.com/ray-project/ray/blob/master/release/serve_tests/workloads/deployment_graph_wide_ensemble.py>`_
+     - Runs 10 node ensemble, constructed with a call graph, that performs basic arithmetic at each node. Ensemble pattern routes the input to 10 different nodes, and their outputs are combined to produce the final output. Simulates 4 clients making 20 requests each.
+     - Head node: AWS EC2 m5.8xlarge. 0 Worker nodes.
+     - * throughput_mean_tps = 8.75
+       * throughput_std_tps = 0.43
+       * latency_mean_ms = 126.15
+       * latency_std_ms = 18.35
+```
+
+:::{note}
+The performance numbers above come from a recent run of the nightly benchmarks.
+:::
+
+<!--- See https://github.com/ray-project/ray/pull/27711 for more context on the benchmarks. -->
+
+Check out [our benchmark workloads'](https://github.com/ray-project/ray/tree/master/release/serve_tests/workloads) source code directly to get a better sense of what they test. You can see which cluster templates each benchmark uses [here](https://github.com/ray-project/ray/blob/8eca6ae852e2d23bcf49680fef6f0384a1b63564/release/release_tests.yaml#L2328-L2576) (under the `cluster_compute` key), and you can see what type of nodes each template spins up [here](https://github.com/ray-project/ray/tree/master/release/serve_tests).
+
 You can check out our [microbenchmark instructions](https://github.com/ray-project/ray/blob/master/python/ray/serve/benchmarks/README.md)
-to benchmark on your hardware.
+to benchmark Ray Serve on your hardware.
 
 ## Debugging performance issues
 
 The performance issue you're most likely to encounter is high latency and/or low throughput for requests.
 
-If you have set up [monitoring](serve-monitoring) with Ray and Ray Serve, you will likely observe the following:
+Once you set up [monitoring](serve-monitoring) with Ray and Ray Serve, these issues may appear as:
 
-- `serve_num_router_requests` is constant while your load increases
-- `serve_deployment_queuing_latency_ms` is spiking up as queries queue up in the background
+- `serve_num_router_requests` staying constant while your load increases
+- `serve_deployment_processing_latency_ms` spiking up as queries queue up in the background
 
 Given these symptoms, there are several ways to fix the issue.
```

Comment:
You suggesting several ways to fix the issue.  But I read only three, and even those are not numbered. Maybe rephrase this as:
"Given these symptoms, there a handful ways to address these issues:

1. Make sure you are using the right hardware and resources
2. Are you providing the GPUs using the (`ray_actor_options={“num_gpus”: 1}`)?
3. Are you using one or more cores (`ray_actor_options={“num_cpus”: 2}`) and setting [`OMP_NUM_THREADS`](serve-omp-num-threads) to increase the performance of your deep learning framework?
4. Consider using `async` methods in your callable. See the section below.
5. Consider batching your requests. See the section below.

Reply:
Sometimes solutions or suggestions offered as list of possibilities is easy on the eye.

Reply:
Good idea– I incorporate a version of that list into the "Debugging performance issues" section. It should be a bit easier to read now.